nidx/src/settings.rs

```diff
@@ -174,7 +174,7 @@ pub struct MergeSettings {
 impl Default for MergeSettings {
     fn default() -> Self {
         Self {
-            max_deletions: 100,
+            max_deletions: 500,
```

Comment:
This was supposed to be 1000, I set it to 100 during testing and forgot. I've left a lower default for now (500) and let's test it out.